### PR TITLE
fixes #358. Refactoring the enable use of URL through install command

### LINKED ISSUE #358: `minimesos install` does not accept URLs, while `minimesosFile` does
It's not possible to provide URL to Marathon JSON for `minimesos install` command. However this is possible in `minimesosFile`

### PR DESCRIPTION
**Pull request by sadovnikov**
This PR enables `./bin/minimesos install --marathonFile https://raw.githubusercontent.com/ContainerSolutions/minimesos/27744df7071fb2d4ab15562bf2654ec37f4d539d/apps/weave-scope.json`

### CODE REVIEW

File: minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java [hunk 1]
```diff
@@ -543,12 +539,56 @@ public static File getHostDir() {
...
-            file = new File(getHostDir(), hostFilePath);
+    /**
+     * Taking either URI or path to a file, returns string with its content
+     *
+     * @param location either absolute URI or path to a file
+     *
+     * @return input stream with location content or null
+     */
+    public static InputStream getSourceStream(String location) {
+
```
Comment (frankscholten) [inline minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java]: Rename to `getJsonInputStream`

**Comment by sadovnikov** [comment]
@frankscholten This method is not JSON specific. It's also used for reading `minimesosFile`. Therefore I renamed it to `getInputStream()`

**Comment by frankscholten** [comment]
@sadovnikov Ok

LGTM

**Comment by remmelt** [comment]
:+1:

**Pull request merged**